schemas/url.yml

```diff
@@ -24,15 +24,25 @@
         Note: The `:` is not part of the scheme.
       example: https
 
-    - name: hostname
-      level: extended
+    - name: domain
```

Comment:
Since a URL's host can either be a domain name or an IP, I agree that `domain` may not work here. Is that what you're referring to?

The correct wording to represent this would actually be `host`, not `hostname`.

So if we go to `url.host`, I would be fine to word it like:

> If 'host' is a domain name, it should be copied to source.domain or destination.domain, whichever is appropriate.

So the domain breakdown could happen there, and domains would be expected in two places instead of 3 (a plus, in my mind).

Reply:
host would contain the port, but it doesn't. hostname is correct here. 

Just leave it `hostname` and we are good.